app.js

```diff
@@ -299,37 +343,47 @@ function processArgs(args, options) {
     .wrap(baseArgv.terminalWidth());
 }
 
-
 function _runServer(argv) {
   const app = express();
   const httpServer = argv.https ?
     https.createServer({ key: argv.httpsPrivateKey, cert: argv.httpsCert }, app) :
     http.createServer(app);
   const blocks = {};
 
-  console.log();
-  console.log('Listener Port:\n\t' + argv.port);
-  console.log('HTTPS Enabled:\n\t' + argv.https);
-  console.log();
-  console.log('[IdP]');
-  console.log();
-  console.log('Issuer URI:\n\t' + argv.issuer);
-  console.log('Sign Response Message:\n\t' + argv.signResponse);
-  console.log('Encrypt Assertion:\n\t' + argv.encryptAssertion);
-  console.log('Authentication Context Class Reference:\n\t' + argv.authnContextClassRef);
-  console.log('Authentication Context Declaration:\n\n' + argv.authnContextDecl);
-  console.log('Default RelayState:\n\t' + argv.relayState);
-  console.log();
-  console.log('[SP]');
-  console.log();
-  console.log('Issuer URI:\n\t' + argv.serviceProviderId);
-  console.log('Audience URI:\n\t' + argv.audience);
-  console.log('ACS URL:\n\t' + argv.acsUrl);
-  console.log('SLO URL:\n\t' + argv.sloUrl);
-  console.log('Trust ACS URL in Request:\n\t' + !argv.disableRequestAcsUrl);
-  console.log();
-
-  console.log();
+  console.log(dedent(chalk`
+    Listener Port:
+      {blue ${argv.port}}
+    HTTPS Enabled:
+      {blue ${argv.https}}
+
+    {bold [{cyan Identity Provider}]}
+
+    Issuer URI:
+      {blue ${argv.issuer}}
+    Sign Response Message:
+      {blue ${argv.signResponse}}
+    Encrypt Assertion:
+      {blue ${argv.encryptAssertion}}
+    Authentication Context Class Reference:
+      {blue ${argv.authnContextClassRef || UNDEFINED_VALUE}}
```

Comment:
I'm using `UNDEFINED_VALUE` as a fallback because I think

```
Authentication Context Declaration:
  None
```

looks better than 

```
Authentication Context Declaration:
  undefined
```

I'm open to using a different fallback, if you don't like "none".

Reply:
👍